api/USB.json

```diff
@@ -87,7 +88,8 @@
               "version_added": "8.0"
             },
             "webview_android": {
-              "version_added": false
+              "version_added": false,
+              "notes": "WebView exposes this interface, but does not support WebUSB. See <a href='https://crbug.com/933055'>Chromium bug 933055</a>."
```

Comment:
No need to add the note for every subfeature. It also becomes inaccurate, since `getDevices` isn't an interface.